### PR TITLE
more detailed uwsgi process checks for keystone and nova placement

### PR DESCRIPTION
**Pull request by jyotty**

### CODE REVIEW

File: roles/keystone/tasks/monitoring.yml [hunk 1]
```diff
@@ -3,8 +3,16 @@
...
 
-- name: keystone process check
+- name: remove simple keystone process check
   sensu_process_check: service=uwsgi
+    state: absent
+  notify: restart sensu-client
+
+- name: keystone process check
+  sensu_check:
+    name: nova_placement_process
```
Comment (nirajdp76) [inline roles/keystone/tasks/monitoring.yml]: why are we adding nova_placement process check in keystone role?

Comment (Xiaohua-Shen) [inline roles/keystone/tasks/monitoring.yml]: use sensu_check here is reasonable, but this sensu_check name should be changed.

**Comment by Xiaohua-Shen** [comment]
retest

**Comment by Xiaohua-Shen** [comment]
retest

**Pull request merged**